packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.CustomMenuList.Example.tsx

```diff
@@ -90,7 +90,7 @@ export class ContextualMenuWithCustomMenuListExample extends React.Component<
     this.setState({ items: ITEMS });
   };
 
-  private _onChange = (newValue: string) => {
+  private _onChange = (ev: any, newValue: string) => {
```

Comment:
We should probably type this properly for good practices and such. Not a blocker.